assets/utils/ytHelper.py

```diff
@@ -0,0 +1,33 @@
+import yt_dlp
+from assets.utils.videoFileHelper import clean_for_directory
+
+def is_valid_youtube_url(url) -> bool:
+    if 'youtube' in url and '/watch?v=' in url:
+        return True
+    else:
+        return False
+
+def is_youtube_playlist(url) -> bool:
+    if '&list=' in url or '/playlist?list=' in url:
```

Comment:
I wonder if it would be better to parse the URL through yt-dlp/pafy/youtube_search_python and check if it's a playlist through the response (pretty sure there's a clean way to do this but haven't double checked). Might be more robust than hardcoding two possible values - but not opposed to doing it this way

Reply:
I think the functionality of checking the playlist through a response separately wouldn't make sense as we are fetching the whole webpage anyways here. It doesn't look like yt-dlp can check this with a function, so I think this is probably the best option.